### ISSUE TITLE
Results not as expected as shown in the paper

### ISSUE THREAD

**Issue by KelestZ** [XiaohangZhan/deocclusion#28]
Thanks for the work and the task is, by all means, a hard task. However, I want to leave a comment (also open to discuss) that the model doesn't do an acceptable job for filling-in content with amodal masks. I show some context-filling results below with their image ID shown on the top left. The original images are from COCOA/VAL2014. Current inpainting models may do better even they know nothing about object layers.

![image](https://user-images.githubusercontent.com/27111467/94062521-693c8e80-fdac-11ea-9af3-6b7d5bacc3e4.png)

![image](https://user-images.githubusercontent.com/27111467/94062387-37c3c300-fdac-11ea-86a5-fe2ee526347d.png)

![image](https://user-images.githubusercontent.com/27111467/94065121-4dd38280-fdb0-11ea-820c-f3b771435804.png)

**Comment by XiaohangZhan** [comment]
Indeed some content completion results are not satisfactory. It depends on the inpainting model we adopted. The only difference of content completion and inpainting is the input. The training method and hyperparams are exactly the same. With the advances of inpainting, the results will be better.
